lib/whitehall_importer/import.rb

```diff
@@ -27,6 +27,8 @@ def call
           )
         end
 
+        ClearLinksetLinks.call(whitehall_document["content_id"])
```

Comment:
This shouldn't happen as part of the import as this is an irreversible change. It'll have to happen in a sync step. We don't yet have a method for that. If you want gold points you could create a sync method on WhitehallImporter that calls this and the resync service?

Reply:
I've added a new `sync` method in WhitehallImported. Will update and fix up.